### PR TITLE
Be able to login in development environment as well

### PR DESCRIPTION
**Pull request by taihaku**
Just fixed to be able to sign in development environment.
Issues: #8 

Please review :bow: 

@k0kubun

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -232,7 +232,11 @@
   # ==> OmniAuth
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
-  config.omniauth :github, ENV['OMNIAUTH_CONSUMER_KEY'], ENV['OMNIAUTH_CONSUMER_SECRET'], scope: 'read:org'
+  if Rails.env == 'development'
```
Comment (k0kubun) [inline config/initializers/devise.rb]: Use `Rails.env.development?`

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -232,7 +232,11 @@
...
+    config.omniauth :github, '4c2d6f040d55517738ed', 'edf1307cf3f6120a2af69010ff5a7faa2b02d977', scope: 'read:org', redirect_url: 'http://localhost:3000/'
```
Comment (k0kubun) [inline config/initializers/devise.rb]: Unify the behavior about `redirect_url` with production.

File: config/initializers/devise.rb (hunk 1 above)
Comment (taihaku) [inline config/initializers/devise.rb]: thx, fixed 0a2eb3b

File: config/initializers/devise.rb (hunk 2 above)
Comment (taihaku) [inline config/initializers/devise.rb]: :+1: 0a2eb3b

**Comment by k0kubun** [comment]
Thank you! :beer:

**Pull request merged**